src/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -81,14 +81,7 @@ protected override void SetAdditionalRequestParameters(OAuth2Client client)
 
                 RequestContext.Logger.Verbose("Refreshing access token...");
                 await base.SendTokenRequestAsync().ConfigureAwait(false);
-
-                if (Response.IdToken == null)
```

Comment:
yes. thats the whole point of openid scope.

---
In reply to: [109345584](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/282#discussion_r109345584) [](ancestors = 109345584)